proto/auditlog.proto

```diff
@@ -57,9 +57,10 @@ enum Action {
   UPDATE_ENCRYPTION_CONFIG = 8;
   LINK_GITHUB_REPO = 9;
   UNLINK_GITHUB_REPO = 10;
-  EXECUTE_CLEAN_WORKFLOW = 11;
+  INVALIDATE_ALL_WORKFLOW_RECYCLED_RUNNERS = 11;
```

Comment:
nit: maybe `INVALIDATE_WORKFLOW_VM_SNAPSHOTS`? and `INVALIDATE_VM_SNAPSHOT` for the one below if that's what it's doing

Reply:
Updated